Pipfile

```diff
@@ -29,6 +29,8 @@ pytest-rerunfailures = "*"
 matplotlib = "*"
 secretstorage = "*"
 pytest-asyncio = "*"
+shapely = "==1.7.0"
+geopandas = "==0.10.0"
```

Comment:
can move these from `dev-packages` to `packages` and run `poetry lock` again.

Reply:
pipenv lock